src/main/java/com/google/plantasy/utils/QuizTimingPropertiesUtils.java

```diff
@@ -27,14 +27,14 @@
 import java.util.logging.Level;
 import java.util.TimeZone;
 import java.text.SimpleDateFormat;
-import java.util.Arrays;
+import com.google.plantasy.utils.UserUtils;
+import com.google.plantasy.utils.Game;
 
 public final class QuizTimingPropertiesUtils {
 
     private static final Logger log = Logger.getLogger(QuizTimingPropertiesUtils.class.getName());
 
-    static final List<String> quiz_questions = Arrays.asList(
```

Comment:
also change this function back to this (I think it should work as long as you also get rid of the second parenthesis at the bottom)